<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleBirdClass</class>
 <widget class="QMainWindow" name="SimpleBirdClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleBird</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabWorkspace">
     <attribute name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Workspace</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabFiles">
     <attribute name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/page.png</normaloff>:/icons/page.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Files</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabCharts">
     <attribute name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/chart.png</normaloff>:/icons/chart.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Charts</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabScripts">
     <attribute name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Scripts</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionChangeWorkspace"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionTest"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Visualisations">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Visualisations</string>
    </property>
    <addaction name="actionSpectrogram"/>
    <addaction name="actionFeatureChart"/>
   </widget>
   <widget class="QMenu" name="menu_Calculate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Calculate</string>
    </property>
    <addaction name="actionCalculateEnergy"/>
    <addaction name="actionCalculatePower"/>
    <addaction name="actionCalculateMFCC"/>
    <addaction name="actionCalculateHFCC"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionChangelog"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionToolbars"/>
    <addaction name="separator"/>
    <addaction name="actionDictionary"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Visualisations"/>
   <addaction name="menu_Calculate"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockConsole">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ConsoleWidget" name="consoleWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionChangeWorkspace"/>
   <addaction name="actionTest"/>
   <addaction name="actionSpectrogram"/>
   <addaction name="actionFeatureChart"/>
   <addaction name="separator"/>
   <addaction name="actionRecognizeCurrentFile"/>
   <addaction name="actionDictionary"/>
   <addaction name="actionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QDockWidget" name="dockProperties">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>188</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="WavePropertiesWidget" name="tblWaveProperties"/>
     </item>
     <item row="1" column="0">
      <widget class="ExtractorPropertiesWidget" name="tblExtractorProperties"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/favorite.png</normaloff>:/icons/favorite.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionSpectrogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/spectrogram.png</normaloff>:/icons/spectrogram.png</iconset>
   </property>
   <property name="text">
    <string>Spectrogram</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionCalculateMFCC">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/calculator.png</normaloff>:/icons/calculator.png</iconset>
   </property>
   <property name="text">
    <string>MFCC</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/process.png</normaloff>:/icons/process.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCalculateEnergy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/calculator.png</normaloff>:/icons/calculator.png</iconset>
   </property>
   <property name="text">
    <string>Energy</string>
   </property>
  </action>
  <action name="actionCalculatePower">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/calculator.png</normaloff>:/icons/calculator.png</iconset>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </action>
  <action name="actionToolbars">
   <property name="text">
    <string>Toolbars</string>
   </property>
  </action>
  <action name="actionFeatureChart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feature_chart.png</normaloff>:/icons/feature_chart.png</iconset>
   </property>
   <property name="text">
    <string>Feature chart</string>
   </property>
  </action>
  <action name="actionChangeWorkspace">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder_add.png</normaloff>:/icons/folder_add.png</iconset>
   </property>
   <property name="text">
    <string>Change workspace directory</string>
   </property>
  </action>
  <action name="actionDictionary">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
   </property>
   <property name="text">
    <string>Dictionary...</string>
   </property>
  </action>
  <action name="actionRecognizeCurrentFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Recognize current file</string>
   </property>
  </action>
  <action name="actionCalculateHFCC">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/calculator.png</normaloff>:/icons/calculator.png</iconset>
   </property>
   <property name="text">
    <string>HFCC</string>
   </property>
   <property name="toolTip">
    <string>HFCC</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>widgets/console/ConsoleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WavePropertiesWidget</class>
   <extends>QTableWidget</extends>
   <header>widgets/WavePropertiesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ExtractorPropertiesWidget</class>
   <extends>QTableWidget</extends>
   <header>widgets/ExtractorPropertiesWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
